doc/release-notes.md

```diff
@@ -37,653 +41,50 @@ through a BIP9 (bit 3) deployment.
-- `previousbits` and `coinbase_payload` fields were added in `getblocktemplate`;
-- `getblocktemplate` now returns an array for masternode payments instead of a single object (miners and mining pools have to upgrade their software to support multiple masternode payees);
-- masternode and superblock payments in `getblocktemplate` show payee scriptPubKey in `script` field in addition to payee address in `payee`;
-- `getblockchaininfo` shows BIP9 deployment progress;
-- `help command subCommand` should give detailed help for subcommands e.g. `help protx list`;
-- `compressed` option in `masternode genkey`;
-- `dumpwallet` shows info about dumped wallet and warns user about security issues;
-- `instantlock` field added in output of `getrawmempool`, `getmempoolancestors`, `getmempooldescendants`, `getmempoolentry`,
-`getrawtransaction`, `decoderawtransaction`, `gettransaction`, `listtransactions`, `listsinceblock`;
-- `addlocked` param added to `getreceivedbyaddress`, `getreceivedbyaccount`, `getbalance`, `sendfrom`, `sendmany`,
-`listreceivedbyaddress`, `listreceivedbyaccount`, `listaccounts`.
-
-There are also new RPC commands:
-- `protx` (`list`, `info`, `diff`, `register`, `register_fund`, `register_prepare`,
-`register_submit`, `update_service`, `update_registrar`, `revoke`);
-- `bls generate`;
-- `setprivatesendrounds`;
-- `setprivatesendamount`.
-
-See `help command` in rpc for more info.
-
-Command-line options
---------------------
-
-New cmd-line options:
-- `masternodeblsprivkey`;
-- `minsporkkeys`;
-- `privatesendsessions`;
-- `zmqpubrawinstantsenddoublespend`;
-- `zmqpubhashinstantsenddoublespend`;
-- `zmqpubhashgovernancevote`;
-- `zmqpubhashgovernanceobject`;
-- `zmqpubrawgovernancevote`;
-- `zmqpubhashgovernanceobject`.
-
-Some of them are Devnet only:
-- `budgetparams`;
-- `minimumdifficultyblocks`;
-- `highsubsidyblocks`;
-- `highsubsidyfactor`.
-
-Few cmd-line options are no longer supported:
-- `instantsenddepth`;
-- `mempoolreplacement`.
-
-See `Help -> Command-line options` in Qt wallet or `dashd --help` for more info.
-
-Lots of refactoring and bug fixes
----------------------------------
-
-A lot of refactoring, code cleanups and other small fixes were done in this release.
+There is a new RPC command `getspecialtxes` which returns an array of special transactions found in the specified
+block with different level of verbosity. Also, various `protx` commands show extended help text for each parameter
+now (instead of referencing `protx register`).
+
+Bug fixes/Other improvements
+----------------------------
+There are few bug fixes in this release:
+- Block size should be calculated correctly for blocks with quorum commitments when constructing new block;
+- Special transactions should be checked for mempool acceptance at the right time (nodes could ban each other
+in some rare cases otherwise);
+- Signature verification and processing of special transactions is decoupled to avoid any potential issues.
 
 0.13.0.0 Change log
```

Comment:
```suggestion
 0.13.1.0 Change log
```

Reply:
Hmm...not sure the suggestion will do what I intended...When I initially created it, it tried to remove a non-existent space at the start of the line...